docs/reference/kubectl/cheatsheet.md

```diff
@@ -122,8 +122,8 @@ $ kubectl get pods --selector=app=cassandra rc -o \
 # Get all running pods in the namespace
 $ kubectl get pods --field-selector=status.phase=Running
 
-# Get ExternalIPs of all nodes
-$ kubectl get nodes -o jsonpath='{.items[*].status.addresses[?(@.type=="ExternalIP")].address}'
+# Get InternalIPs of all nodes
```

Comment:
@dstrebel even if the command doesn't always return a valid external IP address, I think the point wasn't to get an internal IP address, but how to get at the cluster from outside... Rather than changing to a command that isn't as directly useful, add a comment into the cheatsheet explaining that not all Nodes will have a value to return there, depending on your cluster?